atom/renderer/renderer_client_base.cc

```diff
@@ -96,16 +96,14 @@ void RendererClientBase::RenderThreadStarted() {
 #endif
 
 #if defined(OS_MACOSX)
-  // Disable rubber banding by default.
   base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
-  if (!command_line->HasSwitch(switches::kScrollBounce)) {
-    base::ScopedCFTypeRef<CFStringRef> key(
-        base::SysUTF8ToCFStringRef("NSScrollViewRubberbanding"));
-    base::ScopedCFTypeRef<CFStringRef> value(
-        base::SysUTF8ToCFStringRef("false"));
-    CFPreferencesSetAppValue(key, value, kCFPreferencesCurrentApplication);
-    CFPreferencesAppSynchronize(kCFPreferencesCurrentApplication);
-  }
+  bool scroll_bounce = command_line->HasSwitch(switches::kScrollBounce);
+  base::ScopedCFTypeRef<CFStringRef> rubber_bandking_key(
+    base::SysUTF8ToCFStringRef("NSScrollViewRubberbanding"));
+  CFPreferencesSetAppValue(rubber_bandking_key,
+                           scroll_bounce ? kCFBooleanTrue : kCFBooleanFalse,
+                           kCFPreferencesCurrentApplication);
+  CFPreferencesAppSynchronize(kCFPreferencesCurrentApplication);
```

Comment:
Suggestion: Instead of `--scroll-bounce`, the renderer could receive `--enable-scroll-bounce`/`--disable-scroll-bounce` and only set the option if one of these two is passed. If no option is passed, use the default value from the operating system.(For an example of how to use the default OS value, see CEF implementation: https://bitbucket.org/chromiumembedded/cef/src/2f6475c0d89c8b94458b6f7a1146724dbccde09f/libcef/renderer/content_renderer_client.cc?at=master&fileviewer=file-view-default#content_renderer_client.cc-424:438)